profile_specs/authorization_profiles.json

```diff
@@ -0,0 +1,22 @@
+[
+  {
```

Comment:
Let's add to the list a task to evaluate if we can simplify the each gem's profile metadata, we have a lot of files (one for each gem) at this point that would require manual update in multiple places with similar data.

Reply:
Added